_rules/SC2-5-3-label-content-name-mismatch.md

```diff
@@ -31,8 +31,6 @@ This rule applies to any element that has:
 
 The complete [visible text content](#visible-text-content) of the target element either matches or is contained within its [accessible name](#accessible-name).
 
-**Note**: Leading and trailing whitespace and difference in case sensitivity should be ignored.
```

Comment:
@annethyme why did you remove this note? Seems pretty important.

Reply:
Because it's in the definition now, since we were writing the same note in many rules.